spec/classes/create_multiple_ini_settings_spec.rb

```diff
@@ -1,5 +1,24 @@
 require 'spec_helper'
 
 describe 'create_multiple_ini_settings' do
-  it { is_expected.to compile }
+  context 'on a non-Windows platform', :if => !Puppet.features.microsoft_windows? do # rubocop:disable Style/HashSyntax
```

Comment:
The fix for this rubocop error is a tiny change, it is recommended we only disable cops that require a rework of the code. From my understanding it will just be :if => to if: it will make the code in the module more consistent.

Reply:
Fixed